File: 1172.py
lista = []
for i in range(0, 10):
    n = int(input())
    if n == 0 or n < 0:
        lista += [1]
    else:
        lista += [n]
for i in range(0, 10):
    print(f'X[{i}] = {lista[i]}')

File: 1172.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

let lista = [];
for(var i = 0; i < 10; i++){
    let n = Number(lines.shift());
    n === 0 || n < 0 ? lista.push(1) : lista.push(n);
}
for(var l = 0; l < 10; l++){
    console.log(`X[${l}] = ${lista[l]}`);
}
